components/inlinealert/stories/inlinealert.stories.js

```diff
@@ -66,5 +67,25 @@ export default {
 	},
 };
 
-export const Default = Template.bind({});
-Default.args = {};
+export const Default = ({
```

Comment:
I noticed we don't have any coverage for the closable or icon, (docs site)[https://opensource.adobe.com/spectrum-css/inlinealert.html#closable], for the inline alert, what would you think about adding a story for a closable version with an icon and having the wrapping text be displayed in that story?

Reply:
Good catch! It looks like we do have controls for all of the variants, but you're right that we don't have separate story instances for them. I'm hesitant to create more story instances since it will increase our number of Chromatic snapshots, but I'll add another Chromatic-only example for a closeable one with an icon, does that sound ok?